neuralmonkey/writers/plain_text_writer.py

```diff
@@ -30,5 +36,25 @@ def writer(path: str, data: Iterator[Any]) -> None:
     return writer
 
 
+def tokenized_text_writer(encoding: str = "utf-8") -> Writer:
+
```

Comment:
místo mezery docstring, tak jako je to u `t2t_tokenized_text_writer`

Reply:
DONE